app/controllers/ApplicationController.java

```diff
@@ -23,16 +19,12 @@
 public final class ApplicationController extends SunriseController {
 
     @Inject
-    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree, final Configuration configuration) {
-        super(client, categoryTree, configuration);
+    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree,
```

Comment:
Yep, I was thinking of grouping some of the services to reduce the number of parameters, but I wanted to have a clear idea of how many will we eventually have before taking a decision in that direction.